### PR TITLE
Update google_io_extended.md

### PR DESCRIPTION
**Pull request by rohanraarora**
Change I/O 2015 to I/O 2016

### CODE REVIEW

File: gdg_events/google_io_extended.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Google I/O Extended 2015
+# Google I/O Extended 201
```
Comment (friedger) [inline gdg_events/google_io_extended.md]: Why did you remove the last number?

File: gdg_events/google_io_extended.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 
-![](https://events.google.com/io2015/images/io15-color.png)
+![](http://www.androidauthority.com/wp-content/uploads/2016/02/Google-IO-2016.jpg)
```
Comment (friedger) [inline gdg_events/google_io_extended.md]: We shouldn't use hosting capabilities from third parties. You can add an image here as well.

File: gdg_events/google_io_extended.md (hunk 1 above)
Comment (rohanraarora) [inline gdg_events/google_io_extended.md]: Shoot! I missed that. I was changing it to 2016

File: gdg_events/google_io_extended.md (hunk 2 above)
Comment (rohanraarora) [inline gdg_events/google_io_extended.md]: Image added. Merge request sent. Thanks for the suggestions :)

Comment (friedger) [inline gdg_events/google_io_extended.md]: Could you change the url to the uploaded image (https://github.com/gdg-wisdom/wisdom-2016/raw/master/Google-IO-2016.jpg)?

**Comment by rohanraarora** [comment]
Done. You can merge this one now.

**Comment by friedger** [comment]
Actually, this is the correct url https://raw.githubusercontent.com/gdg-wisdom/wisdom-2016/master/Google-IO-2016.jpg
